pkg/policies/opa/rego/docker/docker_from/docker_run/AC_DOCKER_0011.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
create a different folder, that is, docker_run under docker. If command, look in your rego( input.<command>[_] ) then create a corresponding folder under docker folder like docker_<command>. Then place the regos and jsons in that folder.